docs/source/datasets.rst

```diff
@@ -73,6 +71,9 @@ click the link provided in the "Module" column.
 | medleydb_pitch_   | | MedleyDB Pitch    | - audio: 🔑         | - :ref:`f0`               | 103    |
 |                   | | Tracking Subset   | - annotations: ✅   | - :ref:`instruments`      |        |
 +-------------------+---------------------+---------------------+---------------------------+--------+
+| mridangam_stroke_ | Mridangam Stroke    | - audio: ✅         | - :ref:`stroke_name`      | 6977   |
+|                   |                     | - annotations: ✅   | - :ref:`tonic`            |        |
++-------------------+------------------- -+---------------------+---------------------------+--------+
```

Comment:
still not building :( there's a ` ` instead of a `-` near the end of column 2

(you can check this by clicking on the `Details` in the readthedocs test)

Reply:
That's right, found it!! As my table did not throw any error I didn't see it. Fixed now!